tests/integration/telemetry/policy/envoy_ratelimit_test.go

```diff
@@ -79,6 +79,17 @@ func TestLocalRouteSpecificRateLimiting(t *testing.T) {
 		})
 }
 
+func TestLocalRateLimitingServiceAccount(t *testing.T) {
+	framework.
+		NewTest(t).
+		Features("traffic.ratelimit.envoy").
+		Run(func(ctx framework.TestContext) {
+			setupEnvoyFilter(ctx, "testdata/enable_envoy_local_ratelimit_sa.yaml")
```

Comment:
Probably clean up the EnvoyFilter after test finishes? Looking at other test in this package, and seems like they all do not do clean up, and I cannot find clean up is performed anywhere.

Reply:
yeah, this is pretty bad. added a cleanup.